certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,383 @@
+                }
+              },
+              "default": {
+                "groupCode": "your_group_here",
+                "workflow": {
+                  "type": "WEB"
+                },
+                "document": {
+                  "templateCode": "templateSample"
+                }
+              },
+              "required": [
+                "workflow",
+                "document",
+                "groupCode"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
+          },
+          "default": {
+            "description": "default"
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/documents/api/v3/template/list/mine": {
+      "get": {
+        "summary": "Get user's templates",
+        "description": "List all the user's templates",
+        "operationId": "GetUserTemplates",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string",
+                  "description": "Template's code"
+                },
+                "title": {
+                  "type": "string",
+                  "description": "Template's title"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Template's description"
+                },
+                "groups": {
+                  "type": "string",
+                  "description": "List of gropus"
+                },
+                "version": {
+                  "type": "string",
+                  "description": "Template's version"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/documents/api/v3/users/whoiam/": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string",
+                  "description": "User code"
+                },
+                "nationalId": {
+                  "type": "string",
+                  "description": "User nationality"
+                },
+                "email": {
+                  "type": "string",
+                  "description": "User email"
+                },
+                "surname": {
+                  "type": "string",
+                  "description": "User surname"
+                },
+                "rol": {
+                  "type": "string",
+                  "description": "User rol"
+                }
+              }
+            }
+          }
+        },
+        "summary": "WhoIAm",
```

Comment:
@anfuca All the summary and description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)